app/services/execute_search.rb

```diff
@@ -14,11 +14,11 @@ def execute(type, search_criteria)
      when :profiles
        sresult = sservice.profiles(search_criteria)
      when :notes
```

Comment:
Here, are we skipping pages?

Reply:
I'm still curious about this but it doesn't seem that it needs to be resolved in this PR. But curious about your thoughts!

Reply:
Yes, here we are just using notes. But this is how the endpoint was working before:
```
def find_nodes(input, limit = 5, order = :default, type = :natural)
   Node.search(query: input, order: order, type: type, limit: limit)
            .where('type = "note" AND node.status = 1 AND title LIKE ?', '%' + input + '%')
[...]
```
And now we are using `Node.search()`:
```
def find_notes(input, limit = 25, order = :natural, type = :boolean)
   Node.search(query: input, order: order, type: type, limit: limit)
            .where("`node`.`type` = 'note'")
```